content/docs/dvclive/index.md

```diff
@@ -0,0 +1,19 @@
+# dvclive
+
+[dvclive](https://cml.dev) is an open-source python library for monitoring
+[dvclive](https://cml.dev) is an open-source python library for monitoring the
+progress of metrics during training of machine learning projects.
```

Comment:
This needs an update to sidebar.json too:

```suggestion
[dvclive](/doc/dvclive) is an open-source python library for monitoring
the progress of metrics during training of machine learning models.
```

Reply:
> during training of machine learning models

Is this too specific? What's the broader use case?

Reply:
I don't think so? Are you asking whether there are non-ML use cases (not likely), or is there something else you think might be too specific?

Reply:
> Are you asking whether there are non-ML use cases (not likely)

Non model-training use cases.

Reply:
E.g. feature extraction? Or any other stage

Reply:
Good question. I think it's safe to say that dvclive is a narrower product built specifically for model training.